### ISSUE TITLE
[trpc-prisma-websockets-starter] Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined

### ISSUE THREAD

**Issue by niemal** [pingdotgg/uploadthing#275]
### What happened?

I imported `utapi` inside a websocket tRPC route.

### What did you expect to happen?

No errors, normal function.

### Version

5.3.3

### Reproduction Steps

```bash
pnpm create next-app --example https://github.com/trpc/trpc --example-path examples/next-prisma-websockets-starter trpc-prisma-websockets-starter
cd trpc-prisma-websockets-starter
pnpm i
pnpm install uploadthing @uploadthing/react react-dropzone
pnpm run dev
```

Add the following lines of code:
  - `import { utapi } from 'uploadthing/server';`
    on line 11 of `src/server/routers/post.ts`
  - `await utapi.deleteFiles('test');`
    on line 63 of `src/server/routers/post.ts`

### Link to reproduction

https://github.com/niemal/trpc-prisma-websockets-starter

### Relevant log output

```bash
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: No "exports" main defined in /tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/@uploadthing/shared/package.json
    at __node_internal_captureLargerStackTrace (node:internal/errors:490:5)
    at new NodeError (node:internal/errors:399:5)
    at exportsNotFound (node:internal/modules/esm/resolve:361:10)
    at packageExportsResolve (node:internal/modules/esm/resolve:641:13)
    at resolveExports (node:internal/modules/cjs/loader:565:36)
    at Module._findPath (node:internal/modules/cjs/loader:634:31)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1061:27)
    at u.default._resolveFilename (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:1519)
    at Module._load (node:internal/modules/cjs/loader:920:27)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at module (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/chunk-JJ6MOFV6.mjs:164:8)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/chunk-JJ6MOFV6.mjs:438:35)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at <anonymous> (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/chunk-EGTGNUUO.mjs:7:8)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/chunk-EGTGNUUO.mjs:56:25)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at <anonymous> (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/server.mjs:6:8)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/uploadthing@5.3.3_zod@3.0.0/node_modules/uploadthing/dist/server.mjs:270:25)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at <anonymous> (/tmp/test/trpc-prisma-websockets-starter/src/server/routers/post.ts:11:23)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/src/server/routers/post.ts:153:2)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at <anonymous> (/tmp/test/trpc-prisma-websockets-starter/src/server/routers/_app.ts:5:28)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/src/server/routers/_app.ts:24:2)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at Module.require (node:internal/modules/cjs/loader:1141:19)
    at require (node:internal/modules/cjs/helpers:110:18)
    at ws (/tmp/test/trpc-prisma-websockets-starter/src/server/wssDevServer.ts:2:27)
    at Object.<anonymous> (/tmp/test/trpc-prisma-websockets-starter/src/server/wssDevServer.ts:23:2)
    at Module._compile (node:internal/modules/cjs/loader:1254:14)
    at Object.F (/tmp/test/trpc-prisma-websockets-starter/node_modules/.pnpm/@esbuild-kit+cjs-loader@2.4.2/node_modules/@esbuild-kit/cjs-loader/dist/index.js:1:941)
    at Module.load (node:internal/modules/cjs/loader:1117:32)
    at Module._load (node:internal/modules/cjs/loader:958:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:169:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:194:25) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}

Node.js v18.16.0
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by markflorkowski** [comment]
cc @juliusmarminge when you're back from vacation.

**Comment by markflorkowski** [comment]
Should hopefully be resolved in the latest release (5.4.0)
